src/query/exceptions.ts

```diff
@@ -0,0 +1,38 @@
+export class SelectClauseException extends Error {
+  constructor() {
+    super('The SELECT clause does not have a proper structure');
+  }
+}
+
+export class WhereClauseException extends Error {
+  constructor() {
+    super('The WHERE clause does not have a proper structure');
+  }
+}
+export class MultipleQueryTypesException extends Error {
+  constructor(type1: string, type2: string) {
+    super(`You can't combine different query types (ex: an ${type1} with an ${type2})`);
```

Comment:
Consider rephrasing to:
`cannot combine multiple query types (ex: ${type1} with ${type2})`

This is not mandatory, just a suggestion.

Reply:
Fix: Cannot combine multiple query types (ex: ${type1} with ${type2}